app/models/manageiq/providers/amazon/cloud_manager/event_parser.rb

```diff
@@ -1,48 +1,45 @@
 module ManageIQ::Providers::Amazon::CloudManager::EventParser
-  # these have their own EmsEvent classes in automate
-  INSTANCE_EVENTS = %w(
-    AWS_EC2_Instance_CREATE
-    AWS_EC2_Instance_UPDATE
-    AWS_EC2_Instance_DELETE
-    AWS_EC2_Instance_running
-    AWS_EC2_Instance_shutting-down
-    AWS_EC2_Instance_stopped
-  ).to_set.freeze
-
-  def self.map_event_type(aws_event_type)
-    if INSTANCE_EVENTS.include?(aws_event_type)
-      aws_event_type
-    else
-      'ConfigurationItemChangeNotification'
-    end
+  def self.parse_config_event!(event, event_hash)
+    event_hash[:message]                   = event["configurationItemDiff"]
```

Comment:
How about keeping / restricting the number of `eventTypes` at least for aws config?
See, what I don't like is, that we open up the amount of events that are possibly coming from aws.
Now, it's impossible to configure all events in `settings.yml` - and I dont know how you can query the DB via the UI for all events that __are__ in the DB.

That was my rationale in https://github.com/ManageIQ/manageiq-providers-amazon/pull/93

@blomquisg could you please weigh in?